.github/workflows/maven.yml

```diff
@@ -0,0 +1,70 @@
+name: CI
+
+on:
+  push:
+  repository_dispatch:
+    types: network_store_updated
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Download cassandra 4.0-alpha3
```

Comment:
You don't need Cassandra for this service?

Reply:
done